src/docfx/cli/Docfx.cs

```diff
@@ -63,6 +63,7 @@ internal static int Run(string[] args, Package? package = null)
                 RestoreCommand(),
                 BuildCommand(package),
                 ServeCommand(package),
+                ApplyTemplateCommand(),
```

Comment:
Consider this command `docfx build --continue`, this may convey the idea of stopping build in the middle and resume build afterwards.